File: 백준/Silver/2581. 소수/소수.py
m = int(input())
n = int(input())

answer1 = []
for i in range(m,n+1):
    for j in range(2,i+1) :
        if i==j :
            answer1.append(i)
        if i%j == 0 :
            break
if not answer1:
	print(-1)
else:
	print(sum(answer1))
	print(min(answer1))  

File: 백준/Silver/2581. 소수/소수.swift
let M = Int(readLine()!)!
let N = Int(readLine()!)!

var sum = 0
var isPrimeNumber = true
var allPrimeNumber = false
var arr: [Int] = []

for i in M...N {
    if i == 1 {
        isPrimeNumber = false
    }else {
        for j in 2..<i {
            if i % j == 0 {
                isPrimeNumber = false
            }
        }
        
        if isPrimeNumber == true {
            arr.append(i)
            allPrimeNumber = true
        }
    }
    isPrimeNumber = true
}

if allPrimeNumber {
    for k in 0..<arr.count {
        sum += arr[k]
    }
    print("\(sum)")
    print("\(arr.min()!)")
}else {
    print("-1")
}